src/assets/scss/_about-me.scss

```diff
@@ -0,0 +1,80 @@
+//About me section
+
+.about-me {
+    background-color: $background-section-color;
+    height: 38rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    width: 100%;
+    
+    
+
+    &--hi-slider {
```

Comment:
about-me__hi-slider :) 

-- modyfikator
__ element 

Np. Masz stały element typu button i chcesz taki sam tylko biały to robisz button--white i nadpisujesz :)